### PR TITLE
Updating the FDI image to correct discovery issue.

### PR DESCRIPTION
**Pull request by amolpati30**
### Problem Statement
As we now have the latest version, 8.10, the test included an older version, which caused the issue.

### Solution
I have updated the version, and it is now working as expected.

### CODE REVIEW

**Comment by amolpati30** [comment]
"trigger": "test-robottelo"
"pytest": "tests/foreman/cli/test_discoveredhost.py -k 'test_positive_verify_updated_fdi_image'

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7848
Build Status: SUCCESS
PRT Comment: pytest /opt/app-root/src/robottelo/tests/foreman -v -m build_sanity --external-logging
Test Result : ======= 12 passed, 5531 deselected, 5639 warnings in 2081.70s (0:34:41) ========
```

**Comment by shubhamsg199** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_discoveredhost.py -k test_positive_verify_updated_fdi_image

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7849
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/cli/test_discoveredhost.py -k test_positive_verify_updated_fdi_image --external-logging
Test Result : ========== 1 passed, 17 deselected, 26 warnings in 806.69s (0:13:26) ===========
```

File: tests/foreman/cli/test_discoveredhost.py [hunk 1]
```diff
@@ -440,9 +440,9 @@ def test_positive_verify_updated_fdi_image(target_sat):
     target_sat.execute('yum -y --disableplugin=foreman-protector install foreman-discovery-image')
 
     if target_sat.os_version.major == 9:
-        version = '8.9' if is_open('SAT-25275') else str(target_sat.os_version)
+        version = '8.10' if is_open('SAT-25275') else str(target_sat.os_version)
     elif target_sat.os_version.major == 8:
-        version = '8.9' if is_open('SAT-24197') else str(target_sat.os_version)
+        version = '8.10' if is_open('SAT-24197') else str(target_sat.os_version)
```
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_discoveredhost.py:445]:
```suggestion
        version = str(target_sat.os_version)
```

Comment (amolpati30) [reply tests/foreman/cli/test_discoveredhost.py]: Changes has been resolved.

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_discoveredhost.py -k test_positive_verify_updated_fdi_image

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7865
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/cli/test_discoveredhost.py -k test_positive_verify_updated_fdi_image --external-logging
Test Result : ========== 1 passed, 17 deselected, 26 warnings in 736.65s (0:12:16) ===========
```

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**